spec/libraries/permissions_spec.rb

```diff
@@ -128,6 +128,7 @@
 
     it "can see all pictures" do
       is_expected.to be_able_to(:read, Alchemy::Picture)
+      is_expected.to be_able_to(:url, Alchemy::Picture)
```

Comment:
I don't think we need the additional permission if anonymous users cannot read pictures.

Reply:
The admin pictures controller has a `authorize_resource` before filter. We could add extra code in the controller to prevent the check happening in the `url` action and use the `read` permission manually, but this here was way shorter.